packages/core/src/common/utils/_font-face.scss

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the terms of the LICENSE file distributed with this project.
+
+// these implementations borrowed from Bourbon v5, and modified to fit our use cases.
+// changes: removed $asset-pipeline argument, added weight/style params like Bourbon v4.
+// https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/library/_font-face.scss
+// https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/utilities/_font-source-declaration.scss
+
+/// Generates an `@font-face` declaration. You can choose the specific file
```

Comment:
why the triple slashes?

Reply:
I dunno, copy-pasted from bourbon source. I think they use it for doc comments? I'll revert to doubles